opentelemetry-api/src/opentelemetry/trace/propagation/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class ContextKeys:
```

Comment:
Is this class and its class methods necessary? Can't the constants `EXTRACT_SPAN_CONTEXT_KEY` and `SPAN_KEY` just be declared here so that they can be imported elsewhere?

Reply:
this could definitely be removed, i just followed the java context prop implementation here.